lambda-functions/elb-cleanup/config.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+)
+
+// cfg global configuration across the whole
+// services
+var cfg config
+
+// config describes the available configuration
+// of the running service
+type config struct {
+	Debug          bool
+	Region         string
+	ExpirationDays int `mapstructure:"expiration_days"`
+}
+
+// Validate makes sure that the config makes sense
+func (c *config) Validate() error {
+	return nil
```

Comment:
let's do some kind of validation that `expiration_days` is set and `region`

Reply:
pushed the validation for `region`